### PR TITLE
Fixes accessibility defects in Pager component

### PR DESCRIPTION
**Pull request by edleeks87**
# Description
Adds label to input, sets cursor for disabled buttons, removes `withTheme()` HOC from Pager and uses callback instead to identify theme

### CODE REVIEW

File: src/components/pager/pager.component.js [hunk 1]
```diff
@@ -57,6 +56,7 @@ const Pager = (props) => {
         { ...props }
         currentPage={ currentPage }
         setCurrentPage={ setCurrentPage }
+        currentTheme={ setCurrentTheme }
```
Comment (stevebutler2210) [inline src/components/pager/pager.component.js]: Shouldn't this use the getter?

Comment (edleeks87) [reply src/components/pager/pager.component.js]: No it's a callback to allow the navigation sub component to set the theme in the parent, it probably needs a better name for the prop though

Comment (edleeks87) [reply src/components/pager/pager.component.js]: What do you think if I rename the prop to `setcurrentThemeName` and pass in the hook setter as a callback to it?

Comment (stevebutler2210) [reply src/components/pager/pager.component.js]: Sorry, yep gotcha. Yeah that reads a bit more clearly 👍

**Pull request merged**